core/src/components/table/table/table.tsx

```diff
@@ -79,8 +89,38 @@ export class TdsTable {
     });
   }
 
+  /** Returns all selected rows data. */
+  @Method()
+  async getSelectedRows() {
```

Comment:
I think it would make more sense to include the selected rows in the tdsSelect event that is emitted.

I don't believe this method would be needed if the selected rows was included there.

It also seems unnecessary to me to include anything more than a row ID, since its the rows that are selected.

But I saw now that we don't support a row-key. 

I guess we can just leave as is for now (to reach 1.0 faster), and then add in a row-id as well as add the selected row ids to the event - in a later release.

Reply:
So on the tdsSelect event we also do a check and emit the cell-value/cell-key of all the selected rows? This would require a DOM check, which we have tried to stay away from?

Might be a convenience worth doing though, rather than the user running the method we run it internally. But I do feel like we should keep the method aswell for now?

Reply:
@alexandereneroth check the two last commits

Reply:
Yea, nevermind. I feel like checking the dom at all, even in this method, shouldnt be needed. The user already has the data and keeps track of the selected rows 🤔

Reply:
Yea, as long as selected rows can only be visible rows. Maybe we can clarify in the doc comment that it only returns selected rows that are present in the DOM.